webapp/layouts/error.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="error-container">
+    <img class="error-image" :src="image" />
+    <br />
+    <span v-if="error.message === 'This page could not be found'" class="error-message">
+      {{ $t('error-pages.default') }}
+    </span>
+    <span v-else class="error-message">{{ $t(error.message) }}</span>
+    <br />
+    <nuxt-link to="/">{{ $t('error-pages.back-to-index') }}</nuxt-link>
+  </div>
+</template>
+
+<script>
+export default {
+  props: ['error'],
+  computed: {
+    message() {
+      const mapping = {
+        403: 'error-pages.403-default',
+        404: 'error-pages.404-default',
+        500: 'error-pages.500-default',
+        503: 'error-pages.503-default',
+      }
+      return this.error.key || mapping[this.error.statusCode] || 'error-pages.default'
+    },
+    image() {
+      return `/img/svg/errors/error${this.error.statusCode ? this.error.statusCode : '500'}.svg`
```

Comment:
```suggestion
      return `/img/svg/errors/error${this.error.statusCode || '500'}.svg`
```

Reply:
@roschaefer I thought you wanted an image :)